### ISSUE TITLE
Wrong type argument: ellama-session, nil

### ISSUE THREAD

**Issue by mberrueta** [s-kostyaev/ellama#85]
Hello! im using emacs 29 on linux
`GNU Emacs 29.2 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.24.20, cairo version 1.16.0) of 2024-02-27` (With doom if Mathers) 

Im running Ollama with docker (im able to call by curl without any problem)

```
HTTP POST http://localhost:11434/api/generate Headers:[((Content-Type . application/json))] Body:[{
  "model": "zephyr",
  "stream": false,
  "prompt": "Why is the sky blue?"
}]
```

I didn't set any config. 
So if I run 

- `llama-chat` I get `or: Wrong type argument: ellama-session, nil`
- `ellama-get-ollama-local-mode` -> `process-lines-handling-status: Wrong type argument: stringp, nil`
- `llama-provider-select` -> `llama2` and then `ellama-chat` -> `File mode specification error: (wrong-type-argument listp keymap)`

It seems that there are wrong init things?

```
 head ellama.el
;;; ellama.el --- Tool for interacting with LLMs -*- lexical-binding: t -*-

;; Copyright (C) 2023  Free Software Foundation, Inc.

;; Author: Sergey Kostyaev <sskostyaev@gmail.com>
;; URL: http://github.com/s-kostyaev/ellama
;; Keywords: help local tools
;; Package-Requires: ((emacs "28.1") (llm "0.6.0") (spinner "1.7.4") (dash "2.19.1"))
;; Version: 0.8.10
;; SPDX-License-Identifier: GPL-3.0-or-later
```

**Comment by mberrueta** [comment]
btw, removed the package install from doom, and installed manually

so version
```
;;; ellama.el --- Tool for interacting with LLMs -*- lexical-binding: t -*-

;; Copyright (C) 2023  Free Software Foundation, Inc.

;; Author: Sergey Kostyaev <sskostyaev@gmail.com>
;; URL: http://github.com/s-kostyaev/ellama
;; Keywords: help local tools
;; Package-Requires: ((emacs "28.1") (llm "0.6.0") (spinner "1.7.4") (dash "2.19.1"))
;; Version: 0.8.10
;; SPDX-License-Identifier: GPL-3.0-or-later
;; Created: 8th Oct 2023

````

and run:

- `ellama-chat` -> `funcall-interactively: Wrong type argument: ellama-session, nil`

but after restart emacs im not able to call llama any more. (still appear as installed, but the command is not available)

**Comment by mberrueta** [comment]
```
  ellama                         20240225.1803  installed             Tool for interacting with LLMs
  dash                           20240103.1301  dependency            A modern list library for Emacs
  llm                            0.9.1          dependency            Interface to pluggable llm backends
  spinner                        1.7.4          dependency            Add spinners and progress-bars to the mode-line for ongoing operations

```

**Comment by s-kostyaev** [comment]
Hi @mberrueta 

Try to `M-x` `toggle-debug-on-error` and reproduce an error. And paste debugger output here/

**Comment by s-kostyaev** [comment]
@mberrueta Unfortunately I have no idea what's the reason of this issue. Can you still reproduce it?

**Comment by s-kostyaev** [comment]
looks like #144 is the reason of this issue.
@mberrueta fix will be in #144
